meme_contest/949d9650-f57b-493d-83ed-46d336e5eb67.json

```diff
@@ -0,0 +1 @@
+{"id":"949d9650-f57b-493d-83ed-46d336e5eb67","started":"2023-06-26T14:24:32.563Z","ended":null,"contest_channel_id":"1121441890420658197","contest_reaction_id":"<:pepethy:1113195373155070073>","contest_winner":null,"active":true,"posts":{},"start_message_id":"1122895177602826282"}
```

Comment:
this should be gitignored and some fields (e.g. `contest_channel_id`) should really be values in `.env`) :)

Reply:
Right. No idea how this made its way here.

The fields are useful, I designed a system that allows for multiple concurrent meme contests (don't ask why, I was just having my fun haha), hence the necessity of this field.

Reply:
okay, just add `meme_contest` to gitignore and delete that directory and it's all good to me